client/blocks/gdpr-banner/index.jsx

```diff
@@ -53,6 +54,9 @@ class GdprBanner extends Component {
 		if ( cookies.sensitive_pixel_option === 'yes' || cookies.sensitive_pixel_option === 'no' ) {
 			return false;
 		}
+		if ( isMobileApp() ) {
+			return false;
+		}
```

Comment:
Is it right to hide the GDPR banner on mobile web views? do we have a native mobile view that presents the warning?